### DISCUSSION TITLE
Disable tabs in Master dialog when isNewOrDeleted

### DISCUSSION THREAD

**Discussion by rhemasystems** [serenity-is/Serenity#6941] [Help]
I have two types of tabs in my Master dialog
The ones circled pink was created using the html template, I have been able to make them inactive when its a new record using the below code

`Serenity.TabsExtensions.setDisabled(this.tabs, 'Churchlife', this.isNewOrDeleted());
 Serenity.TabsExtensions.setDisabled(this.tabs, 'Visitation', this.isNewOrDeleted());`

 the ones circled red were created using the `TabAttribute` in the Master form but I dont know how to make them also inactive when its a new record.

Please any assistance will be greatly appreciated

![image](https://github.com/serenity-is/Serenity/assets/50826372/32257a30-55c3-42d9-8e81-8ef0bad6c2a9)

**Comment by rhemasystems** [comment]
Please any assistance would be greatly appreciated

**Comment by rhemasystems** [comment]
Please any suggestions will be greatly appreciated

**Comment by wezmag** [comment]
If it's not possible to disable the tabs, why not create another dialog and form with limited fields.
